scripts/test-uma-ODRL-policy.ts

```diff
@@ -0,0 +1,17 @@
+const endpoint = 'http://localhost:4000/uma/policies'
+const client1 = 'https://pod.woutslabbinck.com/profile/card#me';
+const client2 = 'https://pod.example.com/profile/card#me';
+
```

Comment:
could you add some documentation that it requires the odrl authorization server to be running.